Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -19,161 +19,152 @@ namespace Mirage
     [DisallowMultipleComponent]
     public class NetworkSceneManager : MonoBehaviour, INetworkSceneManager
     {
+        #region Fields
+
         static readonly ILogger logger = LogFactory.GetLogger(typeof(NetworkSceneManager));
 
+        [Header("Setup Settings")]
+
         [FormerlySerializedAs("client")]
```

Comment:
we can probably remove these old `FormerlySerializedAs` now, since a lot of this is changing anyway

Reply:
Well if i remove it any old projects will lose references no? I kept the file name and meta data same so people wont lost any data references.